docker/dev.yml

```diff
@@ -27,6 +27,9 @@ x-backend-conf: &backend-conf
   # e.g. when developing with Robotoff or openfoodfacts-query
   extra_hosts:
     - "host.docker.internal:host-gateway"
+    - "auth.openfoodfacts.localhost:host-gateway"
```

Comment:
Note that there's already an alias for `auth.${PRODUCT_OPENER_DOMAIN}` assigned to the nginx `frontend` container @john-gom

Reply:
Hi @hangy , yes I had seen that but I didn't want the Keycloak deployment to depend on Product Opener, so I wanted the Keycloak URLs (and ports) to be the same whether running via Product Opener or just stand-alone.